src/pages/home/Home.sass

```diff
@@ -18,7 +18,7 @@
 .card
   margin: 10px
   padding: 0px 0px 20px 0px
-  height: 1300px
+  height: 100vh
```

Comment:
this change is not in the scope of the task

Reply:
done